libs/cardano-ledger-pretty/src/Cardano/Ledger/Pretty/Conway.hs

```diff
@@ -396,7 +400,7 @@ instance PrettyA (Anchor era) where
   prettyA = viaShow
 
 instance PrettyA (PParamsUpdate era) => PrettyA (GovActionState era) where
-  prettyA gas@(GovActionState _ _ _ _ _ _ _ _ _) =
+  prettyA gas@(GovActionState _ _ _ _ _ _ _ _ _ _) =
```

Comment:
The point of this pattern is so you don't forget to update the implementation as well :smile: 

You need to a pretty  field below